change_rate: 0.00436
elip_a: 2.0
elip_b: 1.0
error_threshold: 2
lead_x: 0
lead_y: 0
lead_z: 1.7
leader_covariance: 0.0005
m_Q_scale: 1
m_R_scale: 6
m_R_scale_leader: 1
m_R_scale_neighbor: 1
m_tao_acc_sqrt: 0.151
m_tao_bias_sqrt: 1.37
m_y_damping_factor: 1.0
m_z_damping_factor: 1.65
neighbor_covariance: 0.005
neighbor_x: 3.3
neighbor_y: 0
neighbor_z: 0.4
precisionRangeErrEst: 100
precisionRangeErrEst_leader: 120
precisionRangeErrEst_neighbor: 80
r_acc: 0.1872
r_h: 0.001
r_pos_l: 0.001
r_pos_n: 0.001
r_vel: 0.3
sigma_r: 0.58
start_x: -0.24
start_y: -0.07
start_z: 0.16
trans_x: -0.237
trans_y: -0.09
yaw_offset: 0
